doc/installation/system/inifile.rst

```diff
@@ -216,3 +216,23 @@ in ``pi.cfg`` using the parameter ``PI_TOKEN_MODULES``:
 
     PI_TOKEN_MODULES = [ "myproject.cooltoken", "myproject.lametoken" ]
 
+.. _custom_web_ui:
+
+Custom Web UI
+-------------
+
+The Web UI is a single page application, that is initiated from the file
+``static/templates/index.html``. This file pulls all CSS, the javascript framework
+and all the javascript business logic.
+
+You can configure privacyIDEA to use a complete different, your own Web UI.
```

Comment:
... completely different Web UI.

Reply:
I needed two steps to make it more clear:
358c881a and 784d97b3